ui/.storybook/README.md

```diff
@@ -0,0 +1,66 @@
+# Vault UI Storybook
+
+This README outlines the details of running and collaborating on the Vault UI Storybook.
+
+## Prerequisites
+
+You will need to have all the dependencies for the [Vault UI](../README.md#prerequisites) installed.
+
+## Commands at a Glance
+
+| Command                                    | Description               |
+| ------------------------------------------ | ------------------------- |
+| `yarn`                                     | install dependencies      |
+| `yarn storybook`                           | run storybook             |
+| `ember generate story [name-of-component]` | generate a new story      |
```

Comment:
Did we try moving this to `component-story`? Did we also look at overriding the default component blueprint so that we get JSDoc starting comments for free?

Reply:
Didn't get to that but can look today